netci.groovy

```diff
@@ -46,13 +46,13 @@ def addPushJob(String project, String branch, String os, String configuration)
     Utilities.addGithubPushTrigger(newJob);
 }
 
-["RHEL7.2", "Windows_NT", "CentOS7.1"].each { os ->
+["Ubuntu14.04", "Ubuntu16.04", "Ubuntu16.10", "Fedora24", "Debian8.4", "RHEL7.2", "Windows_NT", "CentOS7.1", "OSX10.12"].each { os ->
```

Comment:
Just an FYI: [Fedora 24 reached its end of life a few months ago](https://fedoraproject.org/wiki/End_of_life).

Perhaps it would be better to target 26? Perhaps even 27, which is scheduled for release in a few weeks.

Reply:
Thanks Omair, I'll try this out!

Reply:
Ah, looks like we don't have a 26 or 27 image yet.  I'll look in to this.

Reply:
Thanks! Is there a way we can help produce these images for Fedora? I have a list of build-dependencies, for example, here: https://pagure.io/fedora-dotnet/blob/f26/f/dotnet.spec#_14

Reply:
I've added an internal work item for this and put your list in it, thanks Omair.  I'll try to get it looked at soon, for now I'll just go with what we have.